variables.tf

```diff
@@ -303,3 +303,24 @@ variable "cluster_autoscaler_helm_config" {
   description = "Cluster Autoscaler Helm Config"
   default     = { "version" : "9.28.0" }
 }
+
+variable "enable_ecr_pullthrough_rule" {
+  type        = bool
+  description = "Specifies if ECR pull-through rule, and accompanying resources, will be created."
+  default     = false
+}
+
+variable "enable_ecr_pullthrough_secret" {
+  type        = bool
+  description = "Specifies if secret storing ECR pull-through credentials will be created."
+  default     = false
+}
+
+variable "registry_credentials" {
+  type        = map(string)
+  description = "Data to populate ECR pull-through rule's credentials secret with."
+  default = {
+    username    = ""
+    accessToken = ""
```

Comment:
Please use `null` as a default value for keys `username` and `accessToken`

Reply:
OK, it's best practice, and it seems to work just fine with empty value.